portfolio/src/main/webapp/index.html

```diff
@@ -108,26 +110,31 @@ <h3 class="project-name">Orgspace</h3>
 				to connect with other organizations and individuals at Google CSSI in
 				Mountain View in 2019.</p>
 			<div class="space2"></div>
+			<div class="row">
 
-			<!-- Comments Feature -->
-			<h3 class="project-title">Comments</h3>
-			<div class="space1"></div>
-
-			<form class="comment-box" action="/data" method="POST" autocomplete="off" id="sendComment">
-				<input type="text" id = "comment-text" name="comment-text"
-                    placeholder="Comment..." minlength="1" maxlength="120" required/>
-				<!-- </form> -->
-				<input class="comment-button" type="submit" value="Submit Comment" form="sendComment"/> <br/>
-				<button class="comment-button"id="delete-comments" onclick="deleteComments()"> Delete Comments </button>
+				<!-- Comments Feature -->
+				<div class="col">
+					<h3 class="project-title">Comments</h3>
+					<div class="space1"></div>
 
-				<div class="space2"></div>
+					<form class="comment-box" action="/data" method="POST" autocomplete="off" id="sendComment">
```

Comment:
Since form is not a void html element, please include a closing </form> tag as well.